core/blockchain.go

```diff
@@ -1298,10 +1289,14 @@ func (bc *BlockChain) writeBlockWithState(block *types.Block, receipts []*types.
 	}
 	rawdb.WriteBlock(bc.db, block)
 
-	root, err := state.Commit(bc.chainConfig.IsEIP158(block.Number()))
-	if err != nil {
-		return NonStatTy, err
+	root, modifiedAccounts, commitErr := state.Commit(bc.chainConfig.IsEIP158(block.Number()))
+	log.Info("Sending StateChangeEvent to the feed", "block number", block.Number(), "count", len(modifiedAccounts.ModifiedAccounts))
```

Comment:
Totally non-blocking but I wonder if `Trace` or `Debug` might be more appropriate here, assuming this happens pretty frequently

Reply:
Yep, totally! Nice catch.